### DISCUSSION TITLE
Unknown metric: val_loss

### DISCUSSION THREAD

**Discussion by tobias-tengler** [keras-team/autokeras#1535] [Q&A] [answered]
I followed the tutorial linked on your site, regarding text classification: [https://autokeras.com/tutorial/text_classification/](https://autokeras.com/tutorial/text_classification/)
Although I used a different dataset, I get an error that doesn't seem to be related to the dataset.
After the first epoch has been trained and the on_epoch_end callback is executed I get an error: "Unknown metric: val_loss"
Any idea what could be causing this?

**Comment by tobias-tengler** [comment, marked as answer]
The issue was that I was also specifying a non zero-based `initial_epoch`. Solved.
